stan/math/prim/scal/fun/grad_inc_beta.hpp

```diff
@@ -27,8 +28,8 @@ inline void grad_inc_beta(double& g1, double& g2, double a, double b,
   double dF2 = 0;
   if (C)
     grad_2F1(dF1, dF2, a + b, 1.0, a + 1, z);
-  g1 = (c1 - 1.0 / a) * c3 + C * (dF1 + dF2);
-  g2 = c2 * c3 + C * dF1;
+  g1 = fma((c1 - inv(a)), c3, C * (dF1 + dF2));
```

Comment:
this is silly, sorry what's fma?

Reply:
https://en.cppreference.com/w/cpp/numeric/math/fma